File: 1. Two Sum/Solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int c = 1;
        while(true){
            for(int i=0;i<nums.length-c;i++){
                if(nums[i]+nums[c+i]==target){
                    return new int[]{i,i+c};
                }
            }
            c++;
        }
    }
}

File: 1. Two Sum/Solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dt = {}
        for i in range(len(nums)):
            dt[nums[i]] = i
        
        for i in range(len(nums)):
            if dt.get(target-nums[i],False) and i!=dt[target-nums[i]]:
                return (i,dt[target-nums[i]])